### PR TITLE
Create vehicle controller

### PR DESCRIPTION
**Pull request by Prashanttiwari2190**
@abhinavatai Please review

### CODE REVIEW

File: src/SFM.VehicleBuilder.Application/Queries/UXGetModelQuery/UXGetModelQueryHandler.cs [hunk 1]
```diff
@@ -20,7 +21,12 @@ public UXGetModelQueryHandler(IChromeDataService chromeDataService)
 
         public async Task<IEnumerable<Model>> Handle(UXGetModelQuery request, CancellationToken cancellationToken)
         {
-            var models = await chromeDataService.GetModels(request.Year, request.Division);
+            var models = (await chromeDataService.GetModels(request.Year, request.Division)).Select(i =>
+                        new Model
+                        {
+                            ModelId = i.modelId,
+                            ModelName = i.modelName,
+                        }).ToArray();
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Application/Queries/UXGetModelQuery/UXGetModelQueryHandler.cs:29]: use ToList() @Prashanttiwari2190

File: src/SFM.VehicleBuilder.Application/Queries/UXGetMakeQuery/UXGetMakeQueryHandler.cs [hunk 2]
```diff
@@ -20,7 +21,12 @@ public UXGetMakeQueryHandler(IChromeDataService chromeDataService)
 
         public async Task<IEnumerable<Division>> Handle(UXGetMakeQuery request, CancellationToken cancellationToken)
         {
-            var divisions = await chromeDataService.GetDivisions(request.Year);
+            var divisions = (await chromeDataService.GetDivisions(request.Year)).Select(i =>
+                          new Division
+                          {
+                              DivisionId = i.divisionId,
+                              DivisionName = i.divisionName,
+                          }).ToArray();
```
Comment (abhinavatai) [inline src/SFM.VehicleBuilder.Application/Queries/UXGetMakeQuery/UXGetMakeQueryHandler.cs:29]: use ToList() @Prashanttiwari2190 for memory efficiency,

**Review by abhinavatai — CHANGES_REQUESTED** [review]

File: src/SFM.VehicleBuilder.Application/Queries/UXGetMakeQuery/UXGetMakeQueryHandler.cs (hunk 2 above)
Comment (Prashanttiwari2190) [reply src/SFM.VehicleBuilder.Application/Queries/UXGetMakeQuery/UXGetMakeQueryHandler.cs:29]: ok.

File: src/SFM.VehicleBuilder.Application/Queries/UXGetModelQuery/UXGetModelQueryHandler.cs (hunk 1 above)
Comment (Prashanttiwari2190) [reply src/SFM.VehicleBuilder.Application/Queries/UXGetModelQuery/UXGetModelQueryHandler.cs:29]: Done.

**Review by abhinavatai — APPROVED** [review]

**Pull request merged**